### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Qiang-Feng**

### CODE REVIEW

**Comment by lsquires** [comment]
Great moves Qiang, proud of you, keep it up!

File: 112.tex [hunk 1]
```diff
@@ -704,7 +704,7 @@ \subsection{Computer Arithmetic}
 \paragraph{Multiplication}
 
 $a_{1}a_{0}\times b_{1}b_{0}=a_{1}\cdot b_{1}\cdot2^{2}+a_{0}\cdot b_{1}\cdot2+a_{1}\cdot b_{0}\cdot2+a_{0}\cdot b_{0}$.
-Multiplication by 2 is equivalent to a right shift, so for 2 bits:
+Multiplication by 2 is equivalent to a left shift, so for 2 bits:
```
Comment (lsquires) [inline 112.tex:707]: Nice

**Review by lsquires — APPROVED** [review]
10/10
would commit

**Review by jordanspooner — CHANGES_REQUESTED** [review]
I don't think an explanation of adjacent state assignments is necessary. Does it add an extra line? If so, we'll need to reword it again.

**Review by jordanspooner — APPROVED** [review]
Otherwise, all good. Thanks, Qiang!

**Pull request merged**